src/publicodes-state/types.d.ts

```diff
@@ -45,6 +45,7 @@ export type Tutorials = Record<string, boolean>
 export type ComputedResultsFootprint = {
   bilan: number
   categories: Record<DottedName, number>
+  subcategories: Record<DottedName, Record<DottedName, number>>
```

Comment:
Est-ce que les subcategories devraient être sous ce format ? J'ai l'impression que ce serait plus somple de les avoir toutes dans un seul objet (et pas divisées par catégories). Mais je ne suis vraiment pas sur que ça soit une bonne idée

Reply:
Let's go

Reply:
Hmm alors après vérif, il semble que cette structure est déjà employé à pas mal d'endroits du site, partout où les subcategories sont déjà utilisées. Un peu relou de modifier tout, je te propose de rester sur ce format

Reply:
Question bête mais on est sur d'avoir la même structure back et front ?

Reply:
@paulsouche what do you think ?